content/python/virtual_guest.py.md

```diff
@@ -0,0 +1,441 @@
+---
+title: "Working with Virtual Guest"
+description: "A few examples on interacting with Virtual Guest"
+date: "2020-07-09"
+classes: 
+    - "SoftLayer_Virtual_Guest"
+    - "SoftLayer_Account"
+tags:
+    - "virtualservers"
+---
+
+### Create Virtual Server
+
+Creation of computing instances on an account. 
+This method is a simplified alternative to interacting with the ordering system directly.
+
+```python
+import SoftLayer
+import json
+
+USERNAME = 'set me'
+API_KEY = 'set me'
+
+client = SoftLayer.Client(username=USERNAME, api_key=API_KEY)
```

Comment:
For these examples, please remove the `USERNAME` and `API_KEY` bits. Idealls the `SoftLayer.Client()` should be reading from the config file, and makes the examples a bit easier to read in my opinion